azure-cs-botservice/Pulumi.yaml

```diff
@@ -0,0 +1,15 @@
+name: azure-cs-botservice
+description: Creates Azure Bot Service with Application Insights
+runtime: dotnet
+template:
+  config:
+    azure:location: 
+      description: Azure DC region
+      default: North Europe 
+    botName:
+      description: Microsoft Azure Bot Name
+    botApplicationId:
+      description: Microsoft AD Application ID
+    botApplicationPassword:
```

Comment:
Where is this one used?

Reply:
This was an an older version - no longer needed (or bot password)